dali/operators/reader/loader/video_loader.h

```diff
@@ -95,6 +96,13 @@ struct OpenFile {
   AVCodecContext* codec;
 #endif
   av_unique_ptr<AVFormatContext> fmt_ctx_;
+
+  struct VideoFileDesc {
+    FILE *file_stream;
```

Comment:
I suggest adding a destructor which would close those streams - and a `close` function. It may be either here (in which case I'd suggest making this struct standalone) or in the enclosing class.

Reply:
Done